planning/behavior_path_planner/src/scene_module/pull_out/pull_out_module.cpp

```diff
@@ -583,9 +565,9 @@ void PullOutModule::updatePullOutStatus()
   std::vector<Pose> start_pose_candidates = searchPullOutStartPoses();
 
   if (parameters_->search_priority == "efficient_path") {
-    planWithPriorityOnEfficientPath(start_pose_candidates, goal_pose);
+    planWithPriority(start_pose_candidates, goal_pose, true);
```

Comment:
I would like to give not bool but `parameters_->search_priority` for adding other policies in the future.

Reply:
@kosuke55 
I agree with your opinion.
And could you check my chenge in [the commit](da7e98af06be785e508e274feebbaf4b244668ec) matchs with your image?

Reply:
thanks, seems good to me!